From git@z Thu Jan  1 00:00:00 1970
Subject: [PATCH] virtio: Add ID for virtio media
From: Albert Esteve <aesteve@redhat.com>
Date: Tue, 10 Mar 2026 09:40:46 +0100
Message-Id: <20260310-virtio-media-id-v1-1-be211bcf682b@redhat.com>
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 7bit

Add VIRTIO_ID_MEDIA definition for virtio-media.

Signed-off-by: Albert Esteve <aesteve@redhat.com>
---
While the Linux driver for virtio media is currently under active
development and review [1], the VirtIO v1.4 specification has
already been published and oficially defines the media device type [2].

Adding the device ID now allows other components and tools to reference it
before the driver lands, and ensures the kernel headers stay aligned
with the specification.

[1] https://lore.kernel.org/linux-media/20250412-virtio-media-v3-1-97dc94c18398@gmail.com/
[2] https://docs.oasis-open.org/virtio/virtio/v1.4/virtio-v1.4.html#x1-82200022
---
 include/uapi/linux/virtio_ids.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/virtio_ids.h b/include/uapi/linux/virtio_ids.h
index 7aa2eb766..a68e1038d 100644
--- a/include/uapi/linux/virtio_ids.h
+++ b/include/uapi/linux/virtio_ids.h
@@ -68,6 +68,7 @@
 #define VIRTIO_ID_AUDIO_POLICY		39 /* virtio audio policy */
 #define VIRTIO_ID_BT			40 /* virtio bluetooth */
 #define VIRTIO_ID_GPIO			41 /* virtio gpio */
+#define VIRTIO_ID_MEDIA			48 /* virtio media */

 /*
  * Virtio Transitional IDs
